.github/workflows/lint_and_test_cpp.yaml

```diff
@@ -231,7 +231,7 @@ jobs:
           export LD_LIBRARY_PATH=`pwd`/thirdparty/output/lib:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/server
           ulimit -s unlimited
           . ./scripts/config_hdfs.sh
-          ./run.sh test -m ${{ matrix.test_module }}
+          ./run.sh test --onebox_opts "disk_min_available_space_ratio=5" -m ${{ matrix.test_module }}
```

Comment:
Set it to 5 may still be flaky, how about adjust the threadholds in throttle_test to make it easier to trigger 'throttle' and make the tests passed?